static/main.css

```diff
@@ -466,6 +466,12 @@ header.topic {
     font-size: 4rem;
 }
 
+.timestamp {
+    text-align: right;
+    margin-block-start: 0.5em;
+    margin-block-end: -1em;
```

Comment:
I would suggest that we go with `margin-bottom: 0;` here in combination with `margin-top: 0;` on the `header h1`.

Reply:
![Screenshot 2023-05-12 at 15-43-04 East Timor](https://github.com/govdirectory/website/assets/2631719/1b8f6c93-880d-4cb3-b028-5aee1c471a2b)